### DISCUSSION TITLE
How to output a net.vtp file

### DISCUSSION THREAD

**Discussion by littlehandwang94** [PMEAL/OpenPNM#2340] [Q&A]
Dear professor, I just have studied the tutorial “Tutorial 1 - Basic”
https://github.com/PMEAL/OpenPNM/blob/dev/examples/tutorials/intro_to_openpnm_basic.ipynb
After I run all codes of this tutorial in Spyder, there does not exist a vtp file.
Then, I can not use Paraview to see the results.

I just want know how to deal with this. Thank you very much.

**Comment by jgostick** [comment]
There are a few tutorials on how to visualize networks in this folder:  https://github.com/PMEAL/OpenPNM/tree/dev/examples/notebooks/io.  You probably want to ".md" file.

**Reply by littlehandwang94** [reply]
Thank you very much for your reply，sir. Actually, I have read that ".md" file before. But the question is after I run a script, the output file named net.vtp is not created.
